superset/viz.py

```diff
@@ -95,19 +95,21 @@ def __init__(self, datasource, form_data, force=False):
         self.process_metrics()
 
     def process_metrics(self):
-        self.metric_dict = {}
```

Comment:
Can't we just use an `OrderedDict` here? https://docs.python.org/2/library/collections.html#collections.OrderedDict